e2e-tests/portal/cypress/support/commands.js

```diff
@@ -83,6 +83,7 @@ Cypress.Commands.add('renameDeal', require('./portal/renameDeal'));
 Cypress.Commands.add('resetPortalUserStatusAndNumberOfSignInLinks', require('./portal/resetPortalUserStatusAndNumberOfSignInLinks'));
 Cypress.Commands.add('userSetPassword', require('./portal/userSetPassword'));
 Cypress.Commands.add('disablePortalUserByUsername', require('./portal/disablePortalUserByUsername'));
+Cypress.Commands.add('createBssDeal', require('./portal/createBssDeal'));
```

Comment:
Can this be `createBssEwcsDeal` please?

Reply:
Have updated this to be `createBssEwcsDeal`